docs/man-openmpi/man3/MPI_Attr_delete.3.rst

```diff
@@ -51,27 +51,6 @@ Note that use of this routine is *deprecated* as of MPI-2, and was
 *deleted* in MPI-3. Please use :ref:`MPI_Comm_delete_attr`. This function does
 not have a mpi_f08 binding.
 
-Delete attribute from cache by key. This function invokes the attribute
```

Comment:
The commit message on this commit does not seem to match the content of the commit.

Reply:
What do you mean here? Isn't the commit this one

```
doc: removed deprecated function descriptions
    
    Now the descriptions of the deprecated functions
    primarily exists in the favored functions.
```
?

Reply:
What I mean is that this commit also includes changes to seealso, fixes typos, adds fixed-width-font changes, etc.  It's not just about updating the deprecated function description locations.

Reply:
Ok, what about the current description?

Reply:
Perfect; thanks!